src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -239,14 +239,23 @@ protected override async Task<bool> AcceptCommand(MessageId id, string method, J
                             return true;
                         }
 
-                        var start = SourceLocation.Parse(args?["start"] as JObject);
-                        //FIXME support variant where restrictToFunction=true and end is omitted
-                        var end = SourceLocation.Parse(args?["end"] as JObject);
-                        if (start != null && end != null && await GetPossibleBreakpoints(id, start, end, token))
-                            return true;
+                        try
+                        {
+                            var start = SourceLocation.Parse(args?["start"] as JObject);
+                            //FIXME support variant where restrictToFunction=true and end is omitted
+                            var end = SourceLocation.Parse(args?["end"] as JObject);
+                            if (start == null || end == null)
+                                throw new ReturnAsErrorException("Both start, and end need to be specified");
```

Comment:
https://chromedevtools.github.io/devtools-protocol/tot/Debugger/#method-getPossibleBreakpoints end is actually optional.  I don't like returning custom errors on limitations that are ours

(see also https://github.com/mono/mono/pull/19999)

Reply:
Ah, forgot about this one. I can integrate that here.